src/constants.jl

```diff
@@ -1,11 +1,5 @@
 DECLARATIVE_PACKAGES_DIR = Pkg.dir("DeclarativePackages")
-@unix_only begin
-    DEFAULT_PROMPT = "\\e[0;35m\\u@\\h:\\W (playground)> \\e[m"
-end
-@windows_only begin
-    DEFAULT_PROMPT = "playground>"
-end
-
+DEFAULT_PROMPT = @windows ? "playground>" : "\\e[0;35m\\u@\\h:\\W (playground)> \\e[m"
```

Comment:
Definitely a personal preference. Hopefully we can eventually just modify the user's prompt rather than having to specify an entirely new one.